supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/api/GcpBatchRequestFactoryImpl.scala

```diff
@@ -151,6 +148,12 @@ class GcpBatchRequestFactoryImpl()(implicit gcsTransferConfiguration: GcsTransfe
 
     val noAddress = runtimeAttributes.noAddress
 
+    // Determine max runtime for Batch
+    val durationInSeconds: Long = data.gcpBatchParameters.batchAttributes.batchTimeout.toSeconds
```

Comment:
I'd suggest to keep the `FiniteDuration` value until we are forced to convert it to seconds, this is less error-prone and more flexible